server.js

```diff
@@ -20,6 +23,35 @@ app.get('/ping', (req, res) => {
   res.send('This is the About page');
 })
 
+
+// Connection URI
+const uri = 'mongodb+srv://NAYANKUMARRAJ:nkr2580@my-first-cluster.hz1puza.mongodb.net/?retryWrites=true&w=majority';

```

Comment:
Consider moving the MongoDB connection logic to a separate module to improve modularity. This will make the code easier to maintain and test. [medium]